doc/command_line.md

```diff
@@ -130,5 +119,13 @@ Options :
         Disable the copy of resources from source to destination folder.
 
   [-j|--generate-javadoc]
-        Enable the generation of the javadoc.
+        Enable the generation of the javadoc. Deprecated, use enable-comments
+        argument.
+
+  [-c|--enable-comments]
+        Adds all comments of the target source code in the Spoon AST.
```

Comment:
"Adds all code comments in the Spoon AST (Javadoc, line-based comments), rewrites them when pretty-printing"